lib/chat_models/chat_anthropic.ex

```diff
@@ -403,6 +404,21 @@ defmodule LangChain.ChatModels.ChatAnthropic do
       {:ok, %Req.Response{status: 529}} ->
         {:error, LangChainError.exception(type: "overloaded", message: "Overloaded")}
 
+      {:ok, %Req.Response{status: 429} = response} ->
+        {:error,
+         LangChainError.exception(
+           type: "rate_limit_exceeded",
+           message: "Service rate limits are exceeded",
+           original: response
+         )}
```

Comment:
It could be worth wrapping these common errors up in functions (i.e. `LangChainError.rate_limit_exceeded(original: response)` to avoid repeating text in other adapters.